src/test/java/executor/service/service/ChromeDriverInitializerTest.java

```diff
@@ -0,0 +1,119 @@
+package executor.service.service;
+
+import executor.service.config.PropertiesLoader;
+import executor.service.model.ProxyConfigHolderDTO;
+import executor.service.model.ProxyCredentialsDTO;
+import executor.service.model.ProxyNetworkConfigDTO;
+import org.apache.commons.configuration2.Configuration;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.Proxy;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import static executor.service.config.PropertiesLoader.APPLICATION_PROPERTIES;
+import static org.junit.jupiter.api.Assertions.*;
+
+class ChromeDriverInitializerTest {
+
+    private WebDriverInitializer driverInitializer;
+    private Configuration testConfiguration;
+    private ProxyConfigHolderDTO proxyConfigHolder;
+    private WebDriver webDriver;
+
+    @BeforeEach
+    void setup() {
```

Comment:
let's use a common code style
```suggestion
    public void setup() {
```

Reply:
ok, it makes a sense :)